src/gui_common/dialogs/CustomConfirmationDialog.cs

```diff
@@ -3,7 +3,8 @@
 /// <summary>
 ///   A custom reimplementation of ConfirmationDialog and AcceptDialog combined into one.
 /// </summary>
-[Tool]
+
+// [Tool]
```

Comment:
I think these other places need to also link to the github issue so that it is easy to find all the places where that issue occurs.